pkg/proxy/handler.go

```diff
@@ -56,7 +56,7 @@ func (h *Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		modelName = "unknown"
 		log.Printf("error reading model from request body: %v", err)
 		w.WriteHeader(http.StatusBadRequest)
-		w.Write([]byte("Bad request: unable to parse .model from JSON payload"))
+		_, _ = w.Write([]byte("Bad request: unable to parse .model from JSON payload"))
```

Comment:
I this required to make the linter happy?

Reply:
yes, either this way or with a `//nolint` annotation. The rule is that every error is handled or explicit ignored.